src/internal/client/client.go

```diff
@@ -812,7 +812,7 @@ func (c APIClient) DeleteAllEnterprise(ctx context.Context) error {
 func DefaultDialOptions() []grpc.DialOption {
 	return []grpc.DialOption{
 		// Don't return from Dial() until the connection has been established.
-		grpc.WithBlock(), //nolint:staticcheck
+		grpc.WithBlock(), //nolint:SA1019
```

Comment:
Yeah I tried to fix this in my first commit to pachyderm ever.  It's like this so that port-forwarding warnings can be printed and that sort of thing.  (Also look at `pachctl version`'s special case; 1 second of blocking, then reconnect without blocking.  Some historical reason for that, I assume.)